style.css

```diff
@@ -412,12 +412,123 @@ header {
 }
 
 .skills {
-  /* margin-left: 145px;
-  margin-right: 115px; */
   padding-right: 40px;
   margin-top: 12px;
 }
 
 .frameworks {
   padding-left: 12px;
 }
+
+.project1:hover,
+.project2:hover {
+  background-color: #fff5e1;
+  color: #ff6b00;
+  border: 1px solid #ff6b00;
+  cursor: pointer;
+}
+
+#resume:hover {
+  background-color: #daa884;
+  cursor: pointer;
+}
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+}
+
+.contact h1 {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 120px 24px 73px 24px;
+  color: #172b4d;
+  text-align: center;
+  min-height: 216px;
+  justify-content: center;
+}
+
+.form-1 {
+  display: flex;
+  flex-direction: column;
+  margin-left: 1.5rem;
+  margin-right: 1.5rem;
+}
+
+#name {
+  padding: 12px;
+  border: none;
+  margin-bottom: 14px;
+}
+
+#email {
+  padding: 12px;
+  border: none;
+  margin-bottom: 14px;
+  margin-top: 39px;
+}
+
+#textarea-1 {
+  padding: 12px;
+  border: none;
+  margin-bottom: 24px;
+  margin-top: 24px;
+  background-color: #fbf8f7;
+  color: black;
+}
+
+input::placeholder {
+  color: #979493;
+}
+
+#textarea-1::placeholder {
+  color: black;
+  font-weight: 400;
+}
+
+#get-in-touch {
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  background-color: #ff6b00;
+  color: white;
+  margin: 36px auto 124px;
+  padding: 12px;
+  border: none;
```

Comment:
- [ ] I can see you did make a change suggested by the previous reviewer, but it seems to not have taken effect on the design. please review the CSS for this section, to match the design given on Figma.
-  [ ] To make sure you know what the outcome of your code will look like, please use the `inspect` option on the chrome developer tool to view your app with a setting of `375` screen size :

# View
![image](https://user-images.githubusercontent.com/50990925/214768553-34c821d1-aa4b-4d69-a251-7c4d2d16f8f9.png)

# Figma:
![image](https://user-images.githubusercontent.com/50990925/214768599-c35e247e-7b7d-49a3-8950-5cf6d19920ff.png)